crates/holochain_metrics/src/lib.rs

```diff
@@ -159,6 +174,9 @@ impl HolochainMetricsConfig {
         {
             const E_CHILD_SVC: &str = "HOLOCHAIN_INFLUXIVE_CHILD_SVC";
 
+            const E_FILE: &str = "HOLOCHAIN_INFLUXIVE_FILE";
```

Comment:
Should we add a comment for each of these constants to explain what they can be used for?